env/bash

```diff
@@ -121,37 +118,29 @@ elif [[ $PARTITION == "darwin-volta-x86" ]]; then
     export ARTEMIS_SUITE=darwin-gpu
     echo "...setup SUCCEEDED"
 elif [[ $PARTITION == "venado-gh" ]]; then
+    module unload cray-libsci
     module load PrgEnv-gnu
     module load cudatoolkit/24.7_12.5
     module load craype-accel-nvidia90
-    module load cmake
-    #module load cray-hdf5-parallel # Compiler wrappers fail to find hdf5.h 2024/11/8
-    export HDF5_ROOT=/opt/cray/pe/hdf5-parallel/1.14.3.1/gnu/12.3
-    export MPICC=cc
-    export MPICXX=CC
-    export MPIFC=ftn
-    export CC=cc
-    export CXX=CC
-    export FC=ftn
-    export F90=ftn
+    unset OMP_NUM_THREADS
+    unset OMP_PROC_BIND
+    unset OMP_PLACES
```

Comment:
Is something setting those? Our Kokkos doesn't include openmp does it?

Reply:
I copied this over previously without checking that it's necessary; these indeed aren't set, so I removed the `unset` calls.